policyuniverse/__init__.py

```diff
@@ -22,8 +22,8 @@
 
 service_data = json.load(open(service_data_path, "r"))
```

Comment:
I think it would make more sense to update this to `service_data = IAMData()` and keep the signature the same on `build_action_categories_from_service_data()` and `build_service_actions_from_service_data()`. That way we only incur the file read and JSON parsing overhead once on import.

Reply:
Updated